scripts/benchmark_augeas_hosts.sh

```diff
@@ -15,17 +15,17 @@ if [ -z "$1" ]; then
 fi
 
 measure_time() {
-	{ time -f "%e" $1 > /dev/null; } 2>&1
+	{ time -f "%e" "$1" > /dev/null; } 2>&1
```

Comment:
I think this is actually fine. If you look at the calls below, you see that they only use 1 argument:

https://github.com/ElektraInitiative/libelektra/blob/58f862ff7a13e4c1549fd657be7ef712e917c574/scripts/benchmark_augeas_hosts.sh#L32-L34

Reply:
Yes, but I'm pretty sure old code relied on the fact that `$1` will be split up again. AFAIK `time` would otherwise look for an executable called e.g. `kdb ls` instead of calling `kdb` with `ls` and the rest as arguments

Reply:
You are right. However, I think using `"$*"` leads to the same problem:

```sh
time_wrapper() {
  time "$*"
}
time_wrapper "ls -a"
```
reports
```
ls -a: command not found
```

Reply:
With `"$*"` you need to pass the command as separate arguments, so `time_wrapper "ls" "-a"` should work.

Reply:
Yes, of course. But that is not how `measure_time` is called in [scripts/benchmark_augeas_hosts.sh](https://github.com/ElektraInitiative/libelektra/blob/58f862ff7a13e4c1549fd657be7ef712e917c574/scripts/benchmark_augeas_hosts.sh).

Reply:
OK, I now understand that you said this from the very beginning. I think your suggestion is good and I'll change the code.